MvvmCross-Forms/MvvmCross.Forms/Views/Hints/MvxPopPresentationHint.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using MvvmCross.Core.ViewModels;
+
+namespace MvvmCross.Forms.Views.Hints
```

Comment:
If the hint is located in this namespace, and a user has defined all ViewModels in a separate dll from the views, then the hint won't be available.

I guess this kind of makes sense because the hint uses the View type, but then again... I'm not sure if it's a good practise to request presentation changes at a platform level.

Maybe the hint can take a ViewModel type?

Reply:
I agree with Nico here. Hints should probably be platform agnostic so they can be used anywhere no matter whether you use forms or not.